### DISCUSSION TITLE
`system` example code results in error

### DISCUSSION THREAD

**Discussion by gorlak** [premake/premake-core#1901] [General]
Running the example code on [the docs page for system](https://premake.github.io/docs/system/) results in:

`Error: expected string; got table`

Also, it doesn't report the line number, which isn't particularly useful when encountering this.

Are the docs out of date, or maybe it's a bug?

**Comment by nickclark2016** [comment]
I don't think I've ever used system as a table personally, so I'm not sure if it's bug vs out of date docs.  @starkos any thoughts?

**Comment by Cannedfood** [comment]
I'm pretty sure the documentation mixes up "system" and "platform" here.

The correct example would be:
```lua
workspace "MyWorkspace"
   configurations { "Debug", "Release" }
   platform { "Windows", "Unix", "Mac" }

   filter "platform:Windows"
      system "windows"

   filter "platform:Unix"
      system "linux"

   filter "platform:Mac"
      system "macosx"
```
